packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -678,6 +678,17 @@ class FlutterPlugin implements Plugin<Project> {
         project.dependencies.add(configuration, dependency, config)
     }
 
+    // Add a task that can be called on flutter projects that prints the java version used in gradle.
+    // Format of the output of this task is used by other code in flutter to evaluate java version.
+    private static void addTaskForJavaVersion(Project project) {
+        // Warning: The name of this task is used by other code.
+        project.tasks.register('javaVersion') {
```

Comment:
How do you feel about calling it (print/show/get)JavaVersion? And is the warning for possible edits to the name?